### ISSUE TITLE
ThemeEditor exception

### ISSUE THREAD

**Issue by aerwin** [maqetta/maqetta#3161]
--> @aerwin 

@billreed63 says the theme editor stopped updating widgets after my commit 4e2e56ca6e1da2477033e393b375c37ae139ead2 for changing the visible palettes.

I see the following exception when trying to change background of a text field in theme editor... no idea how my changes impact this, but I'll take a look this afternoon...

```
Uncaught TypeError: Cannot read property 'url' of undefined ThemeModifier.js:145
declare.getCommandForStyleChange ThemeModifier.js:145
declare._addCommandsForValue ThemeEditor.js:452
declare._propertiesChange ThemeEditor.js:516
lang.hitch lang.js:373
target.(anonymous function).dispatcher aspect.js:94
on.emit on.js:274
on.emit on.js:324
Evented.emit Evented.js:31
publish topic.js:25
connect.publish connect.js:323
declare._changeValue Cascade.js:300
innerChangeValueFunc Cascade.js:184
declare._onFieldChange Cascade.js:253
lang.hitch lang.js:373
target.(anonymous function).dispatcher aspect.js:94
declare._onChange Background.js:343
declare.startup.connect.dojo.hitch.xref Background.js:269
lang.hitch lang.js:375
lang.hitch lang.js:375
target.(anonymous function).dispatcher aspect.js:94
(anonymous function) _FormWidgetMixin.js:202
lang.hitch lang.js:375
(anonymous function) _WidgetBase.js:1106
lang.hitch 
```

**Comment by aerwin** [comment]
--> @billreed63 

@jhpedemonte @JonFerraiolo 

The net here is Bill noticed the theme editor id was different at some of his `'if` statements. I caused this by removing a duplicate definition of the `ThemeEditor` from `ve.plugin.js` (just the one in `themeEditor.plugin.js` exists now). After a phone chat with Jon and Bill, we agreed this was the right thing to do. So, Bill was going to change his code to look for the new editor id.

**Comment by billreed63** [comment]
fixed with 90667f9cad9aa4384285753c0647788711b31652
